in_toto/matchrule_validators.py

```diff
@@ -25,7 +25,8 @@ def _validate_match_rule(keywords):
 
   if not isinstance(keywords, list):
     raise securesystemslib.exceptions.FormatError(
-        "this matching rule is not a list")
+        "Wrong rule format, must be a list.\n\t"
```

Comment:
Also not part of this PR, but our lab projects should choose a single way to wrap long lines.

I spoke with @awwad about this, but we think indenting wrapped lines looks best.  For example,

```Python
raise FormatError("Argument"
  " is improperly formatted.  Valid"
  " format of argument is X.")
```
Note, that's 2 spaces per indentation.

I've used the following in the past, but it's more difficult to read and edit:

```Python
raise FormatError("Argument is improperly"
                  " formatted.  Valid format"
                  " of argument is X.")
```